tests/cancun/eip4844_blobs/test_blob_txs.py

```diff
@@ -500,43 +589,36 @@ def test_valid_blob_tx_combinations(
     "parent_excess_blobs,parent_blobs,tx_max_fee_per_blob_gas,tx_error",
     [
         # tx max_blob_gas_cost of the transaction is not enough
-        (
+        pytest.param(
             SpecHelpers.get_min_excess_blobs_for_blob_gas_price(2) - 1,  # blob gas price is 1
             SpecHelpers.target_blobs_per_block() + 1,  # blob gas cost increases to 2
             1,  # tx max_blob_gas_cost is 1
             "insufficient max fee per blob gas",
+            id="insufficient_max_fee_per_blob_gas",
         ),
         # tx max_blob_gas_cost of the transaction is zero, which is invalid
-        (
+        pytest.param(
             0,  # blob gas price is 1
             0,  # blob gas cost stays put at 1
             0,  # tx max_blob_gas_cost is 0
             "invalid max fee per blob gas",
+            id="invalid_max_fee_per_blob_gas",
         ),
     ],
-    ids=["insufficient_max_fee_per_blob_gas", "invalid_max_fee_per_blob_gas"],
 )
 @pytest.mark.valid_from("Cancun")
 def test_invalid_tx_max_fee_per_blob_gas(
     blockchain_test: BlockchainTestFiller,
     pre: Dict,
     env: Environment,
     blocks: List[Block],
-    parent_blobs: int,
-    non_zero_blob_gas_used_genesis_block: Block,
 ):
     """
     Reject blocks with invalid blob txs due to:
 
     - tx max_fee_per_blob_gas is barely not enough
     - tx max_fee_per_blob_gas is zero
     """
-    if parent_blobs:
-        pre[TestAddress2] = Account(balance=10**9)
-        blocks.insert(0, non_zero_blob_gas_used_genesis_block)
```

Comment:
My preference would be to keep this here as its specific to this test within the module